### ISSUE TITLE
Issue with installing

### ISSUE THREAD

**Issue by shahin-qoli** [albertoalmagro/spree_it_is_a_present#2]
Hi
I tried to install the extension on spree latest version (4.4.0) but couldn't finish "bundle exec rails g spree_it_is_a_present:install" because of this error  : 
"/bundle/ruby/3.0.0/gems/spree_it_is_a_present-0.0.2/app/controllers/spree_it_is_a_present/spree/checkout_controller_decorator.rb:19:in `<main>': uninitialized constant Spree::CheckoutController (NameError)"

I think there isn't "CheckoutController" in Spree-core Gem right now.

**Comment by albertoalmagro** [comment]
Hi @shahin-qoli , first of all thank you so much for trying out my extension 🙏 

You are right. Since Spree extracted `spree_frontend` and `spree_backend` from `spree`, those should now be dependencies of `spree_it_is_a_present`.

I will do the following:
- Since current version of `spree_it_is_a_present` works with Spree 4.3.x, I will make it depend on `spree` < 4.4.
- I will investigate if it is possible to release a new version of `spree_it_is_a_present` that works with `spree` 4.4 using `spree_frontend` and `spree_backend`.

**Comment by albertoalmagro** [comment]
Status update:
- Step 1 Fix dependency for current version: ✅ Done.
- Step 2 Release new version for Spree 4.4: 🛑 Blocked by https://github.com/spree/spree_legacy_frontend/issues/17

cc @shahin-qoli

**Comment by albertoalmagro** [comment]
Closed via #5

FYI in order to use Spree Rails Frontend with Spree version 4.4.0, I went around https://github.com/spree/spree_rails_frontend/issues/17 in my Spree store by forking Spree Rails Frontend and extending release 4.4.0 with the commit that fixed the initialization issue with the cart. In short, I pointed spree_frontend to https://github.com/albertoalmagro/spree_rails_frontend/tree/4-4-cart-fixed
```
gem 'spree_frontend', github: 'albertoalmagro/spree_rails_frontend', branch: '4-4-cart-fixed'
```

In the following days I will release new compatible versions for Spree 4.5.x and 4.6.x releases.
